src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -46,7 +46,7 @@ public static TransportExtensions<MsmqTransport> TransactionScopeOptions(this Tr
         /// </summary>
         public static FileRoutingTableSettings DistributeMessagesUsingFileBasedEndpointInstanceMapping(this TransportExtensions<MsmqTransport> config, string filePath)
         {
-            return config.UnicastRouting().Mapping.DistributeMessagesUsingFileBasedEndpointInstanceMapping(filePath);
+            return config.Routing().Mapping.DistributeMessagesUsingFileBasedEndpointInstanceMapping(filePath);
```

Comment:
`DistributeMessagesUsingFileBasedEndpointInstanceMapping` doesn't seem to make much sense outside of `UnicastRouting`?